turbinia/processors/mount_local.py

```diff
@@ -30,6 +30,47 @@
 RETRY_MAX = 10
 
 
+def GetDiskSize(source_path):
+  """Gets the size of disk evidence in bytes.
+
+  Tries using blockdev to query the size of block devices, and falls back on
+  filesize for image files.
+
+  Args:
+    source_path(str): the source path of the disk.
+
+  Returns:
+    int: the size of the disk in bytes.
+  """
+  size = None
+
+  if not os.path.exists(source_path):
+    raise TurbiniaException(
```

Comment:
Would this prevent from creating an Evidence object for it to still be processed? Maybe can just return size if the rest of the code can still run properly.

Reply:
I've changed this to log an error and continue, but if the path doesn't exist, it's going to fail anyway.